<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ticket_history</class>
 <widget class="QDialog" name="ticket_history">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>811</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QTableWidget" name="ticket_tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>931</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ticket_comboBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
